tools/keytools/wolfBootKeygenTool.vcxproj

```diff
@@ -0,0 +1,178 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
These projects both still need their intermediate directory updated to build okay. See patch here: https://github.com/dgarske/wolfBoot/commit/8a823be1e50254d4dca7bb3417e803a385798f11

Reply:
cherry-picked 5c66492